hera_pspec/tests/test_pspecdata.py

```diff
@@ -314,6 +314,89 @@ def test_get_Q_alt(self):
         
         # Check for error handling
         nt.assert_raises(ValueError, self.ds.set_Ndlys, vect_length+100)
+    
+    def test_get_Q(self):
```

Comment:
Is there an analytic validation test you can put here?

Reply:
The latest commit checks Q matrix with expected result for isotropic beam.